CMakeLists.txt

```diff
@@ -2,7 +2,6 @@ project (KratosMultiphysics)
 cmake_minimum_required (VERSION 2.8.6)
```

Comment:
Are we sure?, @philbucher said something about a minimal version

Reply:
i did not change this, but for this to work in windows the minimum is much higher